### ISSUE TITLE
PDS Gosec SARIF report snippet text contains line numbers and also additional source parts

### ISSUE THREAD

**Issue by de-jcup** [mercedes-benz/sechub#967]
## Situation

The `SARIF` output generated by `gosec` does contain a snippet text, but the text does contain line number prefixes. Also additional source code lines are added as well and not only the relevant part.

Example output:
```json
"region" : {
            "endColumn" : 14,
            "endLine" : 69,
            "snippet" : {
              "text" : "68: \t\t}\n69: \t\tvar data = template.HTML(v.TmplFile)\n70: \t\tisTmpl := true\n"
            },
            "sourceLanguage" : "go",
            "startColumn" : 14,
            "startLine" : 69
          }
```
So snippet is here : `"68: \t\t}\n69: \t\tvar data = template.HTML(v.TmplFile)\n70: \t\tisTmpl := true\n"`

Reading https://docs.oasis-open.org/sarif/sarif/v2.1.0/csprd01/sarif-v2.1.0-csprd01.html#_Toc10541136

> NOTE: The snippet property has various uses:
> ·         It allows a SARIF viewer to present the contents of the region even if the artifact from which it was taken is not available.
> ·         It also allows an end user examining a SARIF log file to see the relevant content without opening another file.
> ·         It can be used to improve result matching.
> 

I would have expected : `\t\tvar data = template.HTML(v.TmplFile)` or `var data = template.HTML(v.TmplFile)`

We use  the truncated snippet to identify the relevant part for false-positive handling, so it is very important for us to have only the relevant part here.

## Wanted
Relevant code snippet shall not contain line numbers or other source parts

## Solution
Create an issue at github gosec project

**Comment by de-jcup** [comment]
Created https://github.com/securego/gosec/issues/773 to solve the problem/inform the gosec team about the issue.

**Comment by de-jcup** [comment]
Will be fixed with `gosec >2.9.6` :

![image](https://user-images.githubusercontent.com/11644753/154907691-fbfd7cb2-9388-47b0-93d9-e00f84c5ac18.png)

**Comment by Jeeppler** [comment]
The SARIF snippet generated by GoSec v2.10.0 looks like this:

~~~
"snippet": {
	"text": "return template.HTML(query), false //change to out desired output"
},
~~~

This fixes the issue.
